<launch>

  <node name="cluster_counter" type="cluster_counter" pkg="cluster_counter" output="screen">
  </node>

</launch>
